lib/model/security/LoginModel.class.php

```diff
@@ -92,6 +92,7 @@ protected function __construct() {
 			if (isset($_COOKIE['remember'])) {
 				$remember = RememberLoginModel::instance()->verifyToken($_COOKIE['remember']);
 				if ($remember) {
+                    RememberLoginModel::instance()->rememberLogin($remember);
```

Comment:
@qurben gebruikt spaties ipv tabs (ik heb alle code ooit omgezet naar tabs, want je kan zelf instellen of je die als 2 of 4 spaties wilt weergeven)